10x-wdl/docker-10x/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:zesty
+
+USER root
+
+COPY bcl2fastq /usr/bin/bcl2fastq
+
+ADD cellranger-2.0.0.tar.gz /opt/tenx
```

Comment:
Agree, but the cellranger repo is missing some things that are present in the cellranger tarball. Also, we'd have to compile everything in the cellranger repo, and my understanding is that that's pretty difficult.

I'd prefer to at least download the tarball from 10x rather than copying it, but it seems like the download site is generating a fresh access token for you each time you load the page: https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest

Reply:
Gotcha. I thought we weren't actually using any compiled cellranger code though, only python and bash. What's missing from the cellranger repo that we need? Let's not wait for it for this PR but do you think it's worth asking them to add it?

Reply:
What we need to know is how to correctly compile something like this: https://github.com/10XGenomics/cellranger/blob/master/tenkit/lib/go/src/q30count/main.go

I think their build system is complex; they want the cellranger tarball to just work in lots of environments, so I assume there are lots of tricks.

We are using compiled components of cellranger, but only those for which we have the source.